atomic_scanners/pipeline-scanner/pipeline-scanner

```diff
@@ -3,8 +3,8 @@ scanner_name: pipeline-scanner
 image_name: registry.centos.org/pipeline-images/pipeline-scanner
 default_scan: redhat-release
 scans: [
-  { name: redhat-release,
+  { name: pipeline-scanner,
```

Comment:
We have been using pipeline-scanner in image name as well, I think description is providing info about the scanner, IMO, we should keep the name intact and let the description detail about its scope.
OR
We might want to change the name to `yum-update` throughout for image name, dir name for scanner and all other places.